arm-machinelearning/2017-01-01/swagger/webservices.json

```diff
@@ -0,0 +1,1105 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response payload is identical to the response payload that is returned by the GET operation. The response includes the Provisioning State and the Azure-AsyncOperation header. To get the progress of the operation, call GET operation on the URL in Azure-AsyncOperation header field. For more information about Asynchronous Operations, see https://msdn.microsoft.com/en-us/library/mt742920.aspx.",
+            "schema": {
+              "$ref": "#/definitions/WebService"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_Remove",
+        "description": "Deletes the specified web service.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },        
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Note that a 202 status is returned even if the service did not exist."
+          },
+          "204": {
+            "description": "No Content."
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearning/webServices/{webServiceName}/CreateRegionalBlob":{
+      "post": {
+        "tags": [
+          "WebServices"
+        ],
+        "operationId": "WebServices_CreateRegionalWebServiceProperties",
+        "description": "Creates an encrypted credentials parameter blob for the specified region. To get the web service from a region other than the region in which it has been created, you must first call Create Regional Web Services Properties to create a copy of the encrypted credential parameter blob in that region. You only need to do this before the first time that you get the web service in the new region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceNameParameter"
+          },
+          {
+            "in": "query",
+            "name": "region",
+            "type": "string",
+            "description": "The region for which encrypted credential parameters are created.",
+            "required": true
+          },        
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
According to [RP spec](http://sharepoint/sites/AzureUX/Sparta/_layouts/15/WopiFrame2.aspx?sourcedoc={cc855351-793a-4417-a536-ab992218ad55}&action=view&Source=http%3A%2F%2Fsharepoint%2Fsites%2FAzureUX%2FSparta%2FShared%2520Documents%2FForms%2FAllItems%2Easpx&RootFolder%253D%25252Fsites%25252FAzureUX%25252FSparta%25252FShared%252520Documents%25252FSpecs%2526FolderCTID%253D0x012000BE9C61C2BE4C444E8018AB8C82DE4CF9%2526View%253D%257b03665E4B%2DA3AF%2D42A4%2D98C7%2D8336CE259C56%257d=DefaultItemOpen&1=DefaultItemOpen), POST call need to return 202.